pages.id/linux/adduser.md

```diff
@@ -0,0 +1,24 @@
+# adduser
+
+> Utilitas penambahan pengguna.
+> Informasi lebih lanjut: <https://manpages.debian.org/latest/adduser/adduser.html>.
+
+- Buat seorang pengguna baru dengan sebuah direktori home bawaan dan mendesak pengguna untuk mengatur sebuah password:
```

Comment:
```suggestion
- Buat seorang pengguna baru dengan sebuah direktori pangkal/home bawaan dan mendesak pengguna untuk mengatur sebuah kata sandi:
```

The word `pangkal` is used to be consistent with [`cd`](https://github.com/tldr-pages/tldr/blob/main/pages.id/common/cd.md).